scrapy_poet/middleware.py

```diff
@@ -11,7 +10,47 @@
 from twisted.internet.defer import inlineCallbacks, returnValue
 
 from scrapy_poet import utils
-from scrapy_poet.utils import _SCRAPY_PROVIDED_CLASSES
+from scrapy_poet.page_input_providers import PageObjectInputProvider
+from scrapy_poet.utils import _SCRAPY_PROVIDED_CLASSES, load_provider_classes
+
+
+Provider = PageObjectInputProvider
```

Comment:
This is not being used.

Reply:
We could take the chance to rename this class into something more simple, like the assign statement in this line.